client/homebrew/brewRenderer/tutorialPopup/tutorial.jsx

```diff
@@ -0,0 +1,161 @@
+const React = require('react');
+const createClass = require('create-react-class');
+const _ = require('lodash');
+require('./tutorial.less');
+
+const DISMISS_KEY = 'dismiss_tutorial';
+
+const TutorialPopup = createClass({
+    displayName: 'TutorialPopup',
+
+    getInitialState: function () {
+        return {
+            steps: {},
+            currentStep: 'splitPaneStep', // Initialize to the first step
+        };
+    },
+
+    componentDidMount: function () {
+        this.checkTutorial();
+        window.addEventListener('resize', this.checkTutorial);
+    },
+
+    componentWillUnmount: function () {
+        window.removeEventListener('resize', this.checkTutorial);
+    },
+
+    steps: {
+        splitPaneStep: function () {
+            window.localStorage.setItem(
+                'naturalcrit-pane-split',
+                window.innerWidth / 2
+            );
+            return (
+                <div className="content">
+                    <div className="editor">
+                        <span className="explanation editor">
+                            This is the Editor pane, where you edit your
+                            document using Markdown
+                        </span>
+                    </div>
+                    <div className="brew">
+                        <span className="explanation brew">
+                            This is the Brew pane, it displays a preview of the
+                            document
+                        </span>
+                    </div>
+                </div>
+            );
+        },
+        snippetsStep: function () {
+            return (
+                <div className="content">
+                    <i className="fas fa-arrow-left"></i>
+                    <span className="explanation">
+                        This is the Snippet Bar, here you will find pieces of
+                        code to help you write
+                    </span>
+                    <span>
+                        Be sure to check on this every now and again, new
+                        snippets come every once in a while
+                    </span>
+                </div>
+            );
+        },
+        tabsStep: function () {
+            return (
+                <div className="content">
+                    <span className="explanation">
+                        These are the three tabs, Brew, Style and Properties
+                    </span>
+                    <span className="explanation">
+                        You can use each one to modify the text and structure,
+                        style and properties of the document, respecively
```

Comment:
I know it's early in development, but I think you should shoot for complete sentences (meaning just add periods at end of sentences).  Also, on this line you have a typo (`respectively`).    I suggest a slight rephrasing: 

> There are 3 editors:  Brew, Style, and Properties.

> Use each editor tab to edit your brew text (Markdown/HTML), styles (CSS), and properties.